HW20/roles/hw202/tasks/copy.yml

```diff
@@ -0,0 +1,13 @@
+- name: Copy file
+  become: true
+  ansible.builtin.copy:
+    src: "{{  copy_file  }}"
+    dest: "{{  copy_dest  }}"
+
+- name: Add permissions
+  become: true
+  ansible.builtin.file:
```

Comment:
Why not to set permissions under https://docs.ansible.com/ansible/latest/collections/ansible/builtin/copy_module.html

Reply:
Added as an option. To get permission, use the tag